packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/comparators/utils/order-object-properties.util.ts

```diff
@@ -1,10 +1,6 @@
-export function orderObjectProperties<T extends object>(data: T[]): T[];
-
-export function orderObjectProperties<T extends object>(data: T): T;
-
-export function orderObjectProperties<T extends Array<any> | object>(
+export const orderObjectProperties = <T extends object[] | object>(
   data: T,
-): T {
```

Comment:
@thaisguigon We should keep the use of `function` on the server side, only use arrow function is dropping multiple signature capability